pvmismatch/pvmismatch_lib/pvcell.py

```diff
@@ -140,23 +199,35 @@ def Isat1(self):
         """
         _Tstar = self.Tcell ** 3. / self.pvconst.T0 ** 3.  # scaled temperature
         _inv_delta_T = 1. / self.pvconst.T0 - 1. / self.Tcell  # [1/K]
-        _expTstar = np.exp(
-            self.Eg * self.pvconst.q / self.pvconst.k * _inv_delta_T
-        )
+        k_b = self.pvconst.k / self.pvconst.q
+        if self.diode_model == 'pvsyst': # include diode factor
+            _expTstar = np.exp(
+                self.Eg / k_b / self.N1 * _inv_delta_T
+            )
+        elif self.diode_model == 'desoto':
+
+            _expTstar = np.exp(
+                self.Eg_0 / k_b / self.pvconst.T0 - self.Eg / k_b / self.Tcell
+            )
+        else:
+            _expTstar = np.exp(
+                self.Eg / k_b /self.N1 * _inv_delta_T
```

Comment:
I think here need to remove N1 for 2-diode, should just be `self.Eg / k_b * _inv_delta_T` right?